app/webpacker/components/ContactsPage/ContactForm.jsx

```diff
@@ -30,13 +30,20 @@ export default function ContactForm({ loggedInUserData }) {
   const { save, saving } = useSaveAction();
   const [captchaValue, setCaptchaValue] = useState();
   const [captchaError, setCaptchaError] = useState(false);
+  const [contactSuccess, setContactSuccess] = useState(false);
```

Comment:
This is never rest to false. Maybe it should be reset when the user clicks the send button? Or when they edit any form values? Or after x seconds? Probably the first is easiest/best.

Reply:
Done, took the first one :-)